src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final Random random = new Random();
```

Comment:
```suggestion
    private final Random random = new Random();
```

It is better to avoid using `static` https://medium.com/att-israel/should-you-avoid-using-static-ae4b58ca1de5